samples/parameterization/np_cheb_pgen_pargs.py

```diff
@@ -0,0 +1,35 @@
+from maestrowf.datastructures.core import ParameterGenerator
+import numpy as np
+
+
+def chebyshev_dist(var_range, num_pts):
+    r = 0.5*(var_range[1] - var_range[0])
+
+    angles = np.linspace(np.pi, 0.0, num_pts)
+    xpts = r*np.cos(angles) + r
+    ypts = r*np.sin(angles)
+
+    return xpts, ypts
+
+
+def get_custom_generator(env, **kwargs):
+    p_gen = ParameterGenerator()
```

Comment:
So, more of a general example discussion point: now that these are standalone files in the repo and not just examples in the docs, should we go back and add proper docstrings to these functions?  i know bloats the example docs size with non-critical info, but undocumented functions drive me nuts when i run into them..

Reply:
I was going to make the same suggestion haha